doc/lsst.ap.pipe/apdb.rst

```diff
@@ -62,26 +49,13 @@ In Gen 3, this becomes (see :ref:`ap-pipe-pipeline-tutorial` for an explanation
 
 .. warning::
 
-   As in Gen 2, make sure the APDB is created with a configuration consistent with the one used by the pipeline.
+   Make sure the APDB is created with a configuration consistent with the one used by the pipeline.
    Note that the pipeline file given by ``-p`` may include APDB config overrides of its own.
    You can double-check what configuration is being run by calling :command:`pipetask run` with the ``--show config="apdb*"`` argument, though this lists *all* configuration options, including those left at their defaults.
 
-Databases can also be set up using :ref:`config files <command-line-task-config-howto-configfile>`:
-
-.. code-block:: py
-   :caption: myApdbConfig.py
-
-   config.db_url = "sqlite:///databases/apdb.db"
-
-.. prompt:: bash
-
-   make_apdb.py -C myApdbConfig.py
-   ap_pipe.py repo --calib repo/calibs --rerun myrun -C myApPipeConfig.py --id [optional ID to process]
-
```

Comment:
Okay, I was a bit confused because of the command-line-task reference, but this isn't actually a command-line task.  So this link may break in the future.

Reply:
True, but right now those are the _only_ docs at https://pipelines.lsst.io/v/daily/modules/lsst.pipe.base/ that explain how task configs work.